src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1940,6 +1940,9 @@ If such a class is used as a base class and if the deriving class defines a dest
   <data name="WRN_TypeParameterSameAsOuterTypeParameter" xml:space="preserve">
     <value>Type parameter '{0}' has the same name as the type parameter from outer type '{1}'</value>
   </data>
+  <data name="WRN_TypeParameterSameAsParentFunctionParameter" xml:space="preserve">
```

Comment:
The name implies the type parameter matches a parameter (rather than type parameter) name in an outer scope. Consider renaming, perhaps `WRN_TypeParameterSameAsOuterMethodTypeParameter`.

Reply:
Done